### DISCUSSION TITLE
表单字段开启必填验证之后，通过form.setValues设置字段值不会触发修改field.validateStatus值，还是一样error状态

### DISCUSSION THREAD

**Discussion by Big-Bear-2019** [alibaba/formily#2295] [Q&A 2.x] [answered]
![1634098870](https://user-images.githubusercontent.com/52909947/137066743-64e84cc1-ccd0-4d11-8c4b-238c5aaf24d2.jpg)

**Comment by janryWang** [comment, marked as answer]
setValues不会触发校验的，只有validate方法和onInput会触发

**Reply by Big-Bear-2019** [reply]
那请问我要在setValues之后，手动触发一次validate吗？只是这样会触发其他没set值的字段也引发一次验证的吧？或者有没有比较好的方法可以推荐呢？

**Reply by janryWang** [reply]
可以调用clearErrors或者手动setFeedback，具体看文档
